operations/sort_results_edge_attribute.yml

```diff
@@ -0,0 +1,42 @@
+id: sort_results_edge_attribute
+subclass_of: sort_results
+name: sort results (edge attribute)
+description: This operation sorts the results by the given edge attribute. If in ascending order the maximum value of the results edges with the given attribute will be taken while the minimum will be taken for descending order. If the result has no edges with te given attribute it will be listed last. If `max_results` is given the it truncates the results to at most the given value.
```

Comment:
```suggestion
description: This operation sorts the results by the given edge attribute. If in ascending order, the maximum value of the results edges with the given attribute will be taken while the minimum will be taken for descending order. If a result has no edges with the given attribute, it will be listed last. If `max_results` is given, it truncates the results to at most the given value.
```